src/main/java/org/repodriller/domain/CommitContributor.java

```diff
@@ -0,0 +1,104 @@
+package org.repodriller.domain;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+import java.util.TimeZone;
+
+/**
+ * A POJO to track details about a person involved with a commit.
+ */
+public class CommitContributor {
+	public String name;
+	public String email;
+	public Calendar time;
+	public TimeZone tz;
+
+	public CommitContributor(String name, String email, Calendar time) {
+		this(name, email, time, null);
+	}
+
+	public CommitContributor(String name, String email, Calendar time, TimeZone tz) {
+		this.name = name;
+		this.email = email;
+		this.time = time;
+		this.tz = tz;
+	}
+
+	public CommitContributor(CommitContributor c) {
+		this.name = c.name;
+		this.email = c.email;
+		this.time = c.time;
+		this.tz = c.tz;
+	}
+
+	@Override
+	public String toString() {
+		return String.format("%s <%s> at %s",  name, email, new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(time));
+	}
+
+	public Developer asDeveloper() {
+		return new Developer(name, email);
+	}
+
+	@Override
+	public int hashCode() {
```

Comment:
is this the one that intellij generates automatically?

Reply:
No. I didn't realize an IDE could do that.